### PR TITLE
Splited address to multi fields. Double checked mandatory/optional fi…

### PR DESCRIPTION
**Pull request by jasonfungsing**
…elds

### CODE REVIEW

File: frontend/apps/orams/components/ReferralBuilderForms/ReferralBuilderDelegateInformationForm.js [hunk 1]
```diff
@@ -163,11 +159,7 @@ class ReferralBuilderDelegateInformationForm extends BaseForm {
               name="rcm_fax"
               id="rcm_fax"
               htmlFor="rcm_fax"
-              label="Fax"
-              validators={{ required }}
-              messages={{
-                required: 'Fax is required'
-              }}
+              label="Fax (optional)"
```
Comment (qimyi) [inline frontend/apps/orams/components/ReferralBuilderForms/ReferralBuilderDelegateInformationForm.js]: I am so relieved with this change :)

File: frontend/apps/orams/components/ReferralBuilderForms/ReferralBuilderPersonalDetailsForm.js [hunk 2]
```diff
@@ -191,24 +187,99 @@ class ReferralBuilderPersonalDetailsForm extends BaseForm {
...
+              <label htmlFor="employee_home_address_state">
+                <b>State (optional)</b>
+              </label>
+              <Select
+                block
+                onChange={e => this.onChangeDropdown('employee_home_address_state', e)}
+                model={`${model}.employee_home_address_state`}
+                name="employee_home_address_state"
+                id="employee_home_address_state"
+                options={[
```
[the remark this answers is not in the document]
Comment (qimyi) [reply frontend/apps/orams/components/ReferralBuilderForms/ReferralBuilderPersonalDetailsForm.js]: Agree with Bryan. While you're at it, extract all options for other select fields as well.

File: frontend/apps/orams/components/ReferralBuilderForms/ReferralBuilderPersonalDetailsForm.js [hunk 3]
```diff
@@ -191,24 +187,99 @@ class ReferralBuilderPersonalDetailsForm extends BaseForm {
...
+                  {
+                    value: ``,
```
Comment (qimyi) [inline frontend/apps/orams/components/ReferralBuilderForms/ReferralBuilderPersonalDetailsForm.js]: Actually the values don't have to be backticks

File: frontend/apps/orams/components/ReferralBuilderForms/ReferralBuilderDelegateInformationForm.js (hunk 1 above)
Comment (jasonfungsing) [reply frontend/apps/orams/components/ReferralBuilderForms/ReferralBuilderDelegateInformationForm.js]: LOL

File: frontend/apps/orams/components/ReferralBuilderForms/ReferralBuilderPersonalDetailsForm.js [hunk 4]
```diff
@@ -346,7 +381,7 @@ class ReferralBuilderPersonalDetailsForm extends BaseForm {
               name="employee_date_of_birth"
               id="employee_date_of_birth"
               htmlFor="employee_date_of_birth"
-              label="Employee Date Of Birth"
+              label="Employee date of birth"
               validators={{ required }}
```
Comment (bsigney) [inline frontend/apps/orams/components/ReferralBuilderForms/ReferralBuilderPersonalDetailsForm.js]: I would check that this is a valid date as well (there is a `validDate` validator available - may need to refactor as it checks that it is a future date as well).

File: frontend/apps/orams/components/ReferralBuilderForms/ReferralBuilderServiceDetailsForm.js [hunk 5]
```diff
@@ -136,7 +136,7 @@ class ReferralBuilderServiceDetailsForm extends BaseForm {
               name="date_of_injury"
               id="date_of_injury"
               htmlFor="date_of_injury"
-              label="Date of Injury"
+              label="Date of injury"
               validators={{ required }}
```
Comment (bsigney) [inline frontend/apps/orams/components/ReferralBuilderForms/ReferralBuilderServiceDetailsForm.js]: validate date as previous comment

**Comment by bsigney** [comment]
@jasonfungsing please look into broken build

**Pull request merged**